src/app/components/client/toolbar/Toolbar.tsx

```diff
@@ -19,7 +19,9 @@ export const Toolbar = (props: Props) => {
     <nav className={styles.toolbar}>
       <div className={styles.start}>{props.children}</div>
       <div className={styles.end}>
-        <PremiumBadge user={props.user} />
+        {process.env.PREMIUM_ENABLED === "true" && (
```

Comment:
With this being a client component, I don't see this at all anymore - I think we'd need a `NEXT_PUBLIC_` prefix.

Reply:
Updated in https://github.com/mozilla/blurts-server/pull/3347/commits/962145453dbb5eed16044bc20ad449843fe81f17.